examples/features/torch_batch_process/batch_inference/compare_with_core_api/core_api_inference.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import print_function
+
+import json
+import logging
+import math
+import os
+import pathlib
+from typing import Any, Optional
+
+import filelock
+import model
+import torch
+import torch.distributed as dist
+import torchvision as tv
+import torchvision.transforms as transforms
+from torch import nn
+
+import determined as det
+from determined import common, core, pytorch
```

Comment:
don't import `det.common` in any examples.

It should be renamed `det.internal`.  It just hasn't been renamed yet.